tests/util_test.py

```diff
@@ -0,0 +1,220 @@
+
+import os
+import random
+import numpy as np
+from PIL import Image
+import torch
+import open_clip
+import argparse
+
+os.environ['CUDA_VISIBLE_DEVICES'] = ''
+
+def seed_all(seed = 0):
+    torch.backends.cudnn.deterministic = True
+    random.seed(seed)
+    np.random.seed(seed)
+    torch.manual_seed(seed)
+
+def inference_text(model, batches):
+    y = []
+    with torch.no_grad():
+        for x in batches:
+            x = open_clip.tokenize(x)
+            y.append(model.encode_text(x))
+    return torch.stack(y)
+
+def inference_image(model, preprocess_val, batches):
+    y = []
+    with torch.no_grad():
+        for x in batches:
+            x = torch.stack([preprocess_val(img) for img in x])
+            y.append(model.encode_image(x))
+    return torch.stack(y)
+
+def random_image_batch(batch_size, size):
+    h, w = size
+    data = np.random.randint(255, size = (batch_size, h, w, 3), dtype = np.uint8)
+    return [ Image.fromarray(d) for d in data ]
+
+def random_text_batch(batch_size, min_length = 75, max_length = 75):
+    t = open_clip.tokenizer.SimpleTokenizer()
```

Comment:
this need to be changed using get_tokenizer

Reply:
This will not work, as get_tokenizer returns either an HFTokenizer, which i could use to get the vocab,
or just the `tokenize` function which I cannot use to get the vocab.

Reply:
Do you really need the vocab though?

Reply:
I guess it doesn't matter, we can use any tokenizer to generate random data